lib/util.js

```diff
@@ -91,7 +94,7 @@ const util = {
   },
 
   gclientSync: (options = {}) => {
-    runGClient(['sync', '--force', '--nohooks', '--with_branch_heads'], options)
+    runGClient(['sync', '--force', '--nohooks', '--with_branch_heads', '--with_tags'], options)
```

Comment:
there was something weird in the past with `with_tags` that would sometimes checkout the wrong thing. I think it might be fixed now, but not sure.